### DISCUSSION TITLE
Analytics i portalen?

### DISCUSSION THREAD

**Discussion by onordbo** [fremtind/jokul#2862] [Annet]
Jeg hører flere team som begynner å ta i bruk Mixpanel til forenklet analyse av trafikk og bruksmønster. Kan vi slenge oss på bølgen her eller? 🤠 Hadde vært smud å lære litt om portalen blir brukt.

_cc @joms @gbrownlee @wkillerud @piofinn_

**Comment by joms** [comment]
Det er skal bare være å dure på. Om man passer litt på hva man lagrer, kan dere fint unngå cookie consent også
